src/ArrayUtils.php

```diff
@@ -222,8 +222,8 @@ public static function inArray($needle, array $haystack, $strict = false)
      *
      * @template TKey
      * @template TValue
-     * @param  array<TKey, TValue>|Traversable<TKey, TValue> $iterator  The array or Traversable object to convert
-     * @param  bool                                          $recursive Recursively check all nested structures
+     * @param  array<TKey, TValue>|iterable<TKey, TValue> $iterator  The array or Traversable object to convert
```

Comment:
`iterable<TKey, TValue>` also covers `array` - can be simplified

Reply:
ah of course. I just pushed an update